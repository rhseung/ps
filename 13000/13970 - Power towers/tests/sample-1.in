9 10
1 42
2 2 2
4 2 2 2 2
5 2 2 2 2 2
3 3 4 5
3 4 3 6
3 7 6 21
3 937640 767456 981242
10 42 1 17 17 17 17 17 17 17 17
