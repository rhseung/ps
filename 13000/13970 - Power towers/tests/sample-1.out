2
4
6
6
1
4
1
0
2
